schema.prisma

```diff
@@ -628,6 +628,8 @@ enum CampaignFileRole {
   campaignPhoto
   invoice
   document
+  anyCampaignPicture
```

Comment:
In order to be consice let's rename `anyCampaignPicture` to `profilePhoto`
```suggestion
  profilePhoto
```

All other enum values are photo, not picture

The main function is to be profile photo, not any photo